src/main/hook/zero/options.c

```diff
@@ -49,6 +50,14 @@ static const struct util_options_def zerohook_options_def[] = {
         .is_secret_data = false,
         .default_value.i = 0,
     },
+    {
+        .name = ZEROHOOK_OPTIONS_STR_PATCH_GFX_FRAME_LIMIT,
+        .description = "Set a Frame Limit. Useful for systems that don't work with VSYNC. 0 = disabled, 60 = default",
```

Comment:
```suggestion
        .description = "Set a Frame Limit. Useful for systems that don't work with VSYNC. 0 = disabled, 60 = game default",
```

Not the default as in "default config value". Clarify to avoid confusion.